src/main/java/com/javalearn/circle/logic/CircleValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.javalearn.circle.logic;
+import java.util.regex.*;
+
+public class CircleValidator {
+
+    public boolean isCircleDataValid (String line){
+
+        boolean result = false;
+
+        if (Pattern.matches("[0](\\.[0]+)?\\s+.*", line)){
```

Comment:
no clear why you need two checks (constants with proper names may help)

Reply:
I cannot resolve full correct check by one matching check. becourse we need exclude zero-like (0, 0.00, 00.0) string correspond to radius from correct parsed to double string/